src/kbmod/search/LayeredImage.cpp

```diff
@@ -22,6 +22,28 @@ LayeredImage::LayeredImage(std::string path, const PointSpreadFunc& psf) : psf(p
     loadLayers(path);
 }
 
+LayeredImage::LayeredImage(const RawImage& sci, const RawImage& var, const RawImage& msk,
+                           float time, const PointSpreadFunc& psf) : psf(psf), psfSQ(psf) {
+    // Get the dimensions of the science layer and check for consistency with
+    // the other two layers.
+    width = sci.getWidth();
+    height = sci.getHeight();
+    if (width != var.getWidth() or height != var.getHeight())
```

Comment:
Good question. We have not been doing the same doc strings in the C++ code. But we should. I added it to the bindings.cpp file so that it will make it into the documents (adding comments there is one of the pain points of working with pybind11).